plugins/BEdita/API/src/Model/Action/UpdateAssociatedAction.php

```diff
@@ -14,12 +14,14 @@
 namespace BEdita\API\Model\Action;
 
 use BEdita\Core\Model\Action\BaseAction;
+use Cake\Database\Expression\QueryExpression;
 use Cake\Datasource\Exception\RecordNotFoundException;
 use Cake\ORM\Association;
+use Cake\ORM\Association\BelongsToMany;
 use Cake\Utility\Hash;
 
 /**
- * Command to add links between entities.
+ * Command to update links between entities.
```

Comment:
Why in `BEdita/API` and not in `BEdita/Core`? seems quite *core*

Why same name as the abstract `UpdateAssociatedAction` in *core*?